modules/my_preferences/php/module.class.inc

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+namespace LORIS\my_preferences;
+/**
+ * Class module implements the basic LORIS module functionality
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class Module extends \Module
+{
+    /**
+     * {@inheritDoc}
+     *
+     * @param \User $user The user whose access is being checked.
+     *
+     * @return bool whether access is granted
+     */
+    public function hasAccess(\User $user) : bool
+    {
+        return parent::hasAccess($user);
```

Comment:
Shouldn't this always be `true`

Reply:
No it shouldn't.. anonymous users don't have access to my preferences.

But there's no reason to override the method just to return the parent value either.